src/main/java/com/simibubi/create/content/kinetics/chainConveyor/ChainConveyorRenderer.java

```diff
@@ -168,6 +183,18 @@ private void renderChains(ChainConveyorBlockEntity be, PoseStack ms, MultiBuffer
 					.overlay(overlay)
 					.renderInto(ms, buffer.getBuffer(RenderType.cutoutMipped()));
 			}
+			if (frustum != null && !frustum.testLineSegment((float) (stats.start().x - camPos.x), (float) (stats.start().y - camPos.y), (float) (stats.start().z - camPos.z),
```

Comment:
This is pretty clever, though I wonder if the chain pops off the edge of your screen if you get it lined up right